9.4/root/usr/bin/container-entrypoint

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
```

Comment:
@hhorak just want to avoid future questions about that :-) i'm OK with merging this as it is.